File: test/export/pas/04-arreglos.pas
Program sin_titulo;
Type
	SIN_TIPO = Real;
Const
	ARREGLO_MAX = 100;
Var
	a: Array [1..10] Of SIN_TIPO;
	b: Array [1..20,1..30] Of SIN_TIPO;
	c: Array [1..40] Of SIN_TIPO;
	d: Array [1..50,1..5] Of SIN_TIPO;
	e: Array [1..ARREGLO_MAX] Of SIN_TIPO;
	n: Integer;
Begin
	ReadLn(n);
	ReadLn(a[1]);
	a[10] := a[1];
	WriteLn(b[10,10]);
End.

File: test/export/c/04-arreglos.c
#include<stdio.h>
#define ARREGLO_MAX 100
#define SIN_TIPO float
int main() {
	SIN_TIPO a[10];
	SIN_TIPO b[20][30];
	SIN_TIPO c[40];
	SIN_TIPO d[50][5];
	SIN_TIPO e[ARREGLO_MAX];
	int n;
	scanf("%i",&n);
	scanf("%s",a[0]);
	a[9] = a[0];
	printf("%s\n",b[9][9]);
	return 0;
}
